src/main/java/web/PartyInsertServlet.java

```diff
@@ -0,0 +1,150 @@
+package web;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.URLEncoder;
+
+import org.apache.tomcat.util.http.fileupload.IOUtils;
+
+import dao.PartyDao;
+import dao.PartyReqDao;
+import dao.UserPartyAccessDao;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.annotation.MultipartConfig;
+import jakarta.servlet.annotation.WebServlet;
+import jakarta.servlet.http.HttpServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import jakarta.servlet.http.HttpSession;
+import jakarta.servlet.http.Part;
+import util.StringUtils;
+import vo.Category;
+import vo.Party;
+import vo.PartyAccess;
+import vo.PartyReq;
+import vo.User;
+
+@WebServlet(urlPatterns = "/party/insert")
+@MultipartConfig(
+		fileSizeThreshold = 1024*1024*1,
+		maxFileSize = 1024*1024*5,
+		maxRequestSize = 1024*1024*10)
+public class PartyInsertServlet extends HttpServlet {
+	
+	@Override
+	protected void service(HttpServletRequest req, HttpServletResponse res) throws ServletException, IOException {
+		// 세션에서 로그인 아이디 조회하기
+		HttpSession session = req.getSession(false);
+		String loginId = (String) session.getAttribute("loginId");
+		if (loginId == null) {
+			res.sendRedirect("../login-form.jsp?err=req&job=" + URLEncoder.encode("파티 개설", "utf-8"));
+			return;
+		}
+		// 요청 파라미터 조회
+		
+		// 일반 입력필드의 값 조회
+		int catNo = StringUtils.stringToInt(req.getParameter("partyCat"));
+		if (catNo == 0) {
+			res.sendRedirect("form.jsp?err=cat");
+			return;
+		}
+		String name = req.getParameter("partyName");
+		int quota = Integer.parseInt(req.getParameter("partyQuota"));
+		String description = req.getParameter("partyDescription");
+		// 파티 가입 제한 값 조회
+		String reqAge = req.getParameter("reqAge");
+		String birthStart = req.getParameter("birthStart");
+		String birthEnd = req.getParameter("birthEnd");
+		String reqGen = req.getParameter("reqGen");
+		String gender = req.getParameter("gender");
+		// 파티번호 값 조회 및 저장
+		PartyDao partyDao = PartyDao.getInstance();
+		int partyNo = partyDao.getPartySeq();
+		// 첨부파일 입력필드의 처리
+		Part upfilePart = req.getPart("partyImage");
+		
+		String filename = null;
+		if (!upfilePart.getSubmittedFileName().isEmpty()) {
+			int length = upfilePart.getSubmittedFileName().length();
+			filename = System.currentTimeMillis() + upfilePart.getSubmittedFileName().substring(Math.max(0,length-5));
+			// 업로드된 첨부파일을 지정된 폴더에 저장
+			InputStream in = upfilePart.getInputStream();
+			OutputStream out = new FileOutputStream(new File("/Users/seung-gyu/workspace/web-workspace/party/src/main/webapp/resources/thumbnail", filename));
```

Comment:
```suggestion
			// 업로드된 첨부파일을 지정된 폴더에 저장
			ServletContext context = req.getServletContext();
			String uploadPath = context.getRealPath("/resources/thumbnail");
			InputStream in = upfilePart.getInputStream();
			OutputStream out = new FileOutputStream(new File(uploadPath, filename));
```
파일 시스템 말고 애플리케이션의 경로를 이용하는 방법 제안입니다.
Document Base(프로젝트명)을 party라 하면, wtpwebapps\party\resources\thumbnail\에 저장됩니다.
이후 파일 로딩할 곳에서는, Module명도 party라 하면, party\resources\thumbnail\filename으로 접근할 수 있습니다.